daybreak_install.sh

```diff
@@ -0,0 +1,86 @@
+#! /bin/bash
+
+set -e
+
+COMPILER_BACKENDS="gcc clang zig"
+
+function _choose_compiler_backend() {
+	if [ $# -eq 0 ]; then
+		echo "Could not find a supported compiler backend!"
+		echo "Please install and make available on PATH one of the following:"
+		echo ${COMPILER_BACKENDS}
+		exit 1
+	fi
+
+	CC=$1
+	shift
+	echo "Checking for compiler backend ${CC}"
+	if ! command -v ${CC} &> /dev/null; then
+		echo "Missing compiler backend ${CC}"
+		set_compiler $@
+	fi
+}
+
+function bootstrap_install() {
+	DAYBREAK_BOOTSTRAP=${DAYBREAK_SOURCE_FOLDER}/bootstrap/${OS}/daybreak
+	DAYBREAK_OUTPUT=${DAYBREAK_HOME}/bin/daybreak
+	if [[ ${OS} = "windows" ]]; then
```

Comment:
How is a dev expected to run bash directly on windows outside of WSL for this use case to occur?

Reply:
MinGW and Cygwin will work. We need to make a separate PS1 script for windows install: https://github.com/ExoKomodo/Daybreak/issues/187